capstone/src/main/java/com/google/univiz/CollegeScoreCardData/CollegeScoreCardInterface.java

```diff
@@ -0,0 +1,16 @@
+import java.util.List;
+
+interface CollegeScoreCardInterface {
+  /*General Use Functions*/
+  public boolean validateCollegeName(String collegeName);
+  public List<UniversityData> getCollegesData(List<String> collegeNames);
+
+  /*Search Feature Functions*/
+  public List<UniversityData> getCollegeSuggestions(String currName);
```

Comment:
is this a list of the colleges suggested for the autocomplete? or is this the list of colleges that the user selects? if this is not, then i suggest adding a ```public List<UniversityData> getSelectedColleges``` method.

Reply:
these are the suggestions the autocomplete will provide the user; the getCollegesData function gets the list of colleges the user selects

Reply:
nice ok. you dont have to do this but maybe making the method name ```getUserCollegesData``` or ```getSelectedCollegesData``` would make it clearer that this would be a list of the colleges that the user has picked.